docker-compose-rabbitmq.yml

```
@@ -0,0 +1,14 @@
+services:
+  rabbitmq-1:
+    image: rabbitmq:3.13.3-management
+    container_name: rabbitmq-1
+    hostname: rabbitmq-1
+    environment:
+      - RABBITMQ_ERLANG_COOKIE=rabbitmqCookie
+      - RABBITMQ_DEFAULT_USER=admin
+      - RABBITMQ_DEFAULT_PASS=1234
+    ports:
+      - 15672:15672
+      - 5672:5672
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)